spell.ignore

```diff
@@ -0,0 +1,399 @@
+os
+jeos
+logdir
+rtype
+basestring
+tmp
+cleber
+cli
+cmd
+prepost
+unittests
+ascii
+unicode
+fixme
+html
+json
+sysinfo
+pre
+xunit
+params
+argparse
+urls
+whitespace
+shortname
+url
+str
+traceback
+filename
+namespace
+avocadolib
+docstring
+exc
+tuples
+tuple
+klass
+uid
+areis
+todo
+attr
+muxes
+nics
+sys
+stdout
+splitlines
+throbber
+mem
+func
+fmt
+itertable
+iterable
+ui
+subcommands
+subparsers
+utils
+hostname
+username
+autodetect
+pem
+uptime
+apahim
+proc
+sigterm
+subprocess
+sigstop
+sigtstp
+crosa
+ast
+realname
+bool
+autotest
+py
+intree
+ini
+unittesting
+sigsegv
+booleans
+distros
+logf
+gzipped
+basedir
+profiler
+mbligh
+apw
+shadowen
+mytest
+aaa
+myte
+fs
+dirs
+shlex
+unittest
+sleeptest
+postprocess
+gmail
+yaml
+tty
+ete
+http
+pythonhosted
+phylogenetics
+envs
+throuh
+fileobj
+attrs
+hardcoded
+utf
+uri
+libvirt
+hypervisor
+ip
+https
+githubusercontent
+mcepl
+virt
+addr
+xml
+svn
+jenkins
+ci
+dtkit
+junit
+src
+thalesgroup
+xsd
+testsuite
+timestamp
+testcase
+rogram
+multiline
+motd
+abspath
+multiplexation
+dicts
+restclient
+toplevel
+subcommand
+checksum
+installable
+amd
+libexec
+subparser
+metavar
+signalling
+gdb
+init
+chdir
+ldoktor
+msg
+zipfile
+tarfile
+symlinks
+xz
+rw
+hashfile
+stdlib
+englobing
+www
+tldp
+ldp
+nxm
+unix
+strems
+ncpus
+makeopts
+cflags
+cpu
+cpuinfo
+intel
+bs
+wikipedia
+stackoverflow
+pickable
+kwargs
+autodetection
+urlopen
+dst
+gdbserver
+ack
+tcp
+async
+afaik
+retransmission
+multi
+returncode
+onlinedocs
+retransmissions
+io
+repos
+repo
+lbranch
+inits
+userspace
+isoinfo
+filesystem
+cdrkit's
+libcdio's
+umount
+linux
+vnet
+ibm
+submodules
+groupdict
+lsmod
+github
+ramdisk
+vg
+lv
+mb
+lvs
+vfat
+mkfs
+pu
+ypu
+kbytes
+memtotal
+meminfo
+eg
+gb
+numa
+num
+vmstat
+smaps
+pid
+fragement
+buddyinfo
+splited
+mountpoint
+dev
+mtab
+fstype
+pids
+filesystems
+loopback
+unmount
+umounting
+ro
+progname
+sudo
+fifo
+sig
+ppid
+subshell
+prepended
+sigkill
+rc
+prerun
+expr
+rwx
+rx
+contet
+xinetd
+systemctl
+symlinking
+symlink
+sshd
+vsftpd
+dgram
+rsync
+systemd
+sysctl
+bugzilla
+cgi
+setattr
+exe
+cmdline
+lldpad
+crond
+dbus
+runlevel
+initializer
+ntpd
+httpd
+condrestart
+br
+rcvalle
+pname
+gdebi
+backends
+backend
+zypper
+dpkg
+ipython
+wildcard
+dnf
+suse
+debian
+ubuntu
+unpickable
+sublicense
+noninfringement
+laub
+phed
+nl
+const
+un
+cacm
+ph
+toucheth
+nonterminal
+sym
+startswith
+nonkernel
+nk
+hacky
+nonkernel's
+nonterminals
+inlines
+ptr
+inlined
+goto
+pstate
+lhs
+pptr
+elif
+isnullable
+rhs
+automagically
+preorder
+subtree
+failtest
+failtests
+psutil
+vgs
+selftests
+lvm
+workflow
+paramiko
+ecdsa
+rsa
+dss
+queryable
+unhashable
+safeloader
+loggables
+kosme
+noncharacters
+repr
+len
+hw
+scsi
+asci
+stacktrace
+exps
+umask
+hardcode
+decompressdir
+usa
+simplewarning
+typestring
+pparent
+staticmethods
+noinspection
+chargen
+de
+brattain
+ross
+breakpoint
+si
+sourceware
+harisrir
+jadmanski
+ba
+ag
+meth
+arcname
+escapingsection
+asdf
```

Comment:
Oh, me to blame again. It's probably suppose to be foo/bar/baz, what do you think? Or is it acceptable?